packages/core/src/App/Containers/AccountSwitcherWallet/account-switcher-wallet.scss

```diff
@@ -13,10 +13,15 @@
     }
 
     &__looking-for-cfds {
+        cursor: pointer;
         display: flex;
         justify-content: space-between;
         align-items: center;
         padding: 1.6rem;
+
+        &:hover {
+            text-decoration: underline;
```

Comment:
not necessary but you can place the cursor: pointer within the hover bracket. It makes more sense because it's already there so might aswell 🤷🏼

Reply:
Updated